src/ieee80211_station.erl

```diff
@@ -1736,21 +1736,22 @@ handle_session_timer_ev({_, Level, _} = Ev, {Interval, _, _Opts} = Timer,
 
 % FIXME Atoms are allowed for test cases. Test cases should, however,
 % use DTLS and not plain UDP.
-add_location(ACAtom, Map) when is_atom(ACAtom) ->
-    ?LOG(debug, "Getting location through atom: ~p", [ACAtom]),
-    add_location(atom_to_binary(ACAtom, utf8), Map);
-add_location(AC, Map) when is_binary(AC) ->
+add_location(AC, Map) when is_pid(AC) ->
     ?LOG(debug, "Getting location through: ~p", [AC]),
     try capwap_ac:get_info(AC) of
 	{ok, #{id := Name}} ->
 	    ?LOG(debug, "Getting location for name: ~p", [Name]),
-	    AVPName = capwap_config:get(ac, location_avp, undefined),
-	    case capwap_loc_provider:get_loc(Name, false) of
+	    BinName = case Name of _ when is_binary(Name) -> Name;
+				   _ when is_atom(Name) -> atom_to_binary(Name, utf8);
+				   _ -> <<"undefined">> end,
+	    ?LOG(debug, "Getting location for name: ~p", [{Name, BinName}]),
```

Comment:
Same log at line 1743. Please or remove this one and extend at line 1743 or remove log at line 1743

Reply:
Not same really. TL;DR it was there to check whether we did in fact receive binaries (we won't when plain UDP is used and no certificate is present). I'll remove the first log nonetheless.